MetaMask/metamask-extension
# feat: Add metametrics event to switch modal confirmation

## **Description**

Adds a tracking event to Network Switcher component

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/24283?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/2420

## **Manual testing steps**

1. Go to Uniswap, connect with Metamask
2. Go to PancakeSwap, connect with Metamask
3. Use PancakeSwap to change the network
4. When clicking on "Switch network", the metametrics event should be triggered

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
